packages/gensx/src/utils/bundler.ts

```diff
@@ -42,7 +42,7 @@ export async function bundleWorkflow(
         "run",
         "--rm",
         "--platform",
-        "linux/amd64",
+        "linux/x86_64",
```

Comment:
ah, I suspected this might be incorrect?

Reply:
I believe they are actually the same, but just to be sure.

Reply:
<img width="668" alt="image" src="https://github.com/user-attachments/assets/83467f1a-95ee-4f6f-a569-3f8f4fb7edcb" />